package.json

```diff
@@ -11,29 +11,31 @@
     "lib/"
   ],
   "scripts": {
-    "prepare": "rimraf lib && npm run build",
+    "prebuild": "rimraf lib",
     "build": "tsc",
     "postversion": "git push && git push --tags",
```

Comment:
we should remove this as well.. sorry, just noticed

Reply:
Sorry, I meant the `postversion` script, but Github misplaced my comment. That's fine, I did it https://github.com/pagopa/io-ts-commons/pull/343/commits/9395dec1ddf1564051dcd7b76397e018b9cfcad2